app/helpers/proposals_helper.rb

```diff
@@ -37,6 +37,10 @@ def render_votes_bar(votes_count)
     end
   end
 
+  def last_comment?(comments, index)
+    comments.size == index + 1
```

Comment:
I wonder if `.size` here will return the array size or will it trigger a `count` query? 🤔

Reply:
No idea. Why the concern? Any guess how to break it?

Reply:
just checked, `.size` won't execute a count query as `@comments` is already an array of comments at that time.

Reply:
So that's fine. I just want to check if it is the last record. Do you see any possible improvement here?